### PR TITLE
chore(runtime): upgrade netif crate

### PR DESCRIPTION
**Pull request by bnoordhuis**
v0.1.3 contains code that will stop working with newer versions of libstd because the layout of some std::net types changed.

Refs: https://github.com/bnoordhuis/netif/pull/10

<hr>

The linked PR is the only code change relative to v0.1.3.

### CODE REVIEW

**Review by bartlomieju — APPROVED** [review]

**Comment by bartlomieju** [comment]
Oooops, I somehow got green CI check when loading this PR. Make sure to update `Cargo.lock` before merging.

**Comment by bnoordhuis** [comment]
Oops, done.

**Review by littledivy — APPROVED** [review]

**Comment by kt3k** [comment]
#15914 seems upgraded `netif` to v0.1.4 in `Cargo.lock` without changing `runtime/Cargo.toml`.

And now I see the ipv4 addresses in `Deno.networkInterfaces()` in the reversed order compared to the previous version. Is this expected change?

```
$ deno --unstable      
Deno 1.25.3+a4a894f
exit using ctrl+d or close()
> Deno.networkInterfaces()
[
  {
    family: "IPv4",
    name: "lo0",
    address: "1.0.0.127",
    netmask: "0.0.0.255",
    scopeid: null,
    cidr: "1.0.0.127/8",
    mac: "00:00:00:00:00:00"
  },
...
```

**Comment by bnoordhuis** [comment]
🤦 - I don't know how that slipped through, I even eye-balled the output before release. Anyway, fixed and new version released.

**Comment by kt3k** [comment]
@bnoordhuis Thank you for the quick fix & release!

**Review by kt3k — APPROVED** [review]
LGTM

**Review by cjihrig — APPROVED** [review]

**Comment by cjihrig** [comment]
Merging to unblock all of the PRs in std.

**Pull request merged**
